omod/src/main/webapp/resources/jsonforms/003-HMIS_ACP_ARTCard_RegimenChange.json

```diff
@@ -222,14 +222,15 @@
               "validators": []
             },
             {
-              "label": "Current Line",
-              "type": "programState",
-              "id": "currentLine",
+              "label": "Current Regimen Line",
+              "type": "obs",
+              "id": "currentRegimenLine",
               "readonly": true,
               "questionOptions": {
-                "concept": "",
+                "concept": "80148344-ce80-42f7-9d4a-8c1935efe76e",
+                "rendering": "select",
                 "calculate": {
-                  "calculateExpression": "resolve(api.getLatestObs(patient.id, 'e6a72c98-1c79-4e76-84b1-2db63d27bce6'))?.valueCodeableConcept?.coding[0]?.code"
+                  "calculateExpression": "resolve(api.getLatestObs(patient.id, '80148344-ce80-42f7-9d4a-8c1935efe76e')).then(obs => obs.valueCodeableConcept?.coding[0]?.code)"
```

Comment:
use the uuid for concept 164515 instead of 164350 in the calculated expression.

Reply:
This has been adjusted, and what happens to the New Regimen Line which is using the same concept uuid as shown below:

` "label": "New Regimen Line",
              "type": "obs",
              "id": "newRegimenLine",
              "readonly": true,
              "questionOptions": {
                "concept": "164515AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
                "rendering": "select",
                "calculate": {
                  "calculateExpression": "resolve(api.getLatestObs(patient.id, '164515AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')).then(obs => obs.valueCodeableConcept?.coding[0]?.code)"
                }`